### PR TITLE
Make contour thickness setable

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR follows on from #2168 by fixing a the interior colors of the labels and making the contour thicnkess settable.

It still needs some tests, but is ready for a first look cc @jni @Czaki @kir0ul 

Screen Shot 2021-02-22 at 10 41 19 PM

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by jni — APPROVED** [review]
Some of the specific choices (maximum spinbox value 😂) are questionable, but overall this does the trick and works, so 👍 from me

File: napari/layers/labels/labels.py [hunk 1]
```diff
@@ -664,15 +665,19 @@ def _raw_to_displayed(self, raw):
         else:
             raise ValueError("Unsupported Color Mode")
 
-        if self.contour:
+        if self.contour > 0:
```
Comment (Czaki) [inline napari/layers/labels/labels.py]: I see a big delay when switching to 3D view and no difference in rendering. Maybe disable this in 3d mode? Until 3D section is implemented?

Comment (sofroniewn) [reply napari/layers/labels/labels.py]: done

**Comment by sofroniewn** [comment]
Ok, I've updated this so that contours mode with 3D rendering doesn't have any effect, but emits a warning, which I think makes sense. I've also reduced the max value of the spin box to 1024 in pixel space (easy to change later too).

Once tests pass I can let this sit for a little in case anyone has additional comments and then merge tomorrow

**Pull request merged**